### PR TITLE
feat: add syndicate titles png, svg, and webp sigils

### PR DESCRIPTION
**Pull request by SlayerOrnstein**
### What did you fix? 
Adds sigils for syndicate titles in png, svg, and webp flavors. plus a script to hopefully make it easier to turn future sigils into svg

---

### Reproduction steps

---

### Considerations
- Does this contain a new dependency? **Yes**
- Does this introduce opinionated data formatting or manual data entry? **Yes**
- Does this pr include updated data files in a separate commit that can be reverted for a clean code-only pr? **[es**
- Have I run the linter? **No**
- Is is a bug fix, feature request, or enhancement? **Feature**

### CODE REVIEW

**Comment by SlayerOrnstein** [comment]
I organized the structure of the png, svgs, and webp. I also cleaned up and optimized svgs using https://github.com/svg/svgo

```
entrati.svg:
Done in 36 ms!
30.668 KiB - 1% = 30.37 KiB

archwing.svg:
Done in 3 ms!
1.77 KiB - 0.4% = 1.763 KiB

arbitrations.svg:
Done in 29 ms!
7.294 KiB - 0% = 7.293 KiB

hexis.svg:
Done in 3 ms!
3.438 KiB - 0.2% = 3.432 KiB

loading.svg:
Done in 2 ms!
1.107 KiB - 0% = 1.107 KiB

nightwave.svg:
Done in 19 ms!
41.19 KiB - 0.4% = 41.022 KiB

simaris.svg:
Done in 2 ms!
3.864 KiB - 0% = 3.864 KiB

nightmare.svg:
Done in 4 ms!
5.189 KiB - 0% = 5.189 KiB

sp-logo.svg:
Done in 4 ms!
1.312 KiB - 0% = 1.312 KiB

ostron.svg:
Done in 4 ms!
3.675 KiB - 1.4% = 3.624 KiB

standing.svg:
Done in 6 ms!
2.571 KiB - 0.3% = 2.563 KiB

solaris.svg:
Done in 6 ms!
10.123 KiB - 1.3% = 9.993 KiB

wfcd.svg:
Done in 2 ms!
1.639 KiB - 0% = 1.639 KiB

grineer.svg:
Done in 2 ms!
0.935 KiB - 32.4% = 0.632 KiB

corrupted.svg:
Done in 1 ms!
1.513 KiB - 25% = 1.135 KiB

infested.svg:
Done in 2 ms!
1.58 KiB - 22.6% = 1.224 KiB

corpus.svg:
Done in 1 ms!
1.341 KiB - 21.3% = 1.055 KiB

narmer.svg:
Done in 4 ms!
1.732 KiB - 17.6% = 1.427 KiB

1.svg:
Done in 5 ms!
4.493 KiB - 27.4% = 3.263 KiB

sentient.svg:
Done in 2 ms!
2.382 KiB - 25.7% = 1.77 KiB

5.svg:
Done in 13 ms!
13.846 KiB - 24.4% = 10.47 KiB

narmer2.svg:
Done in 4 ms!
1.732 KiB - 17.6% = 1.427 KiB

4.svg:
Done in 7 ms!
6.85 KiB - 31.4% = 4.698 KiB

3.svg:
Done in 2 ms!
4.384 KiB - 29.1% = 3.108 KiB

NegativeRank1.svg:
Done in 3 ms!
2.113 KiB - 18.1% = 1.73 KiB

ArbitersLevel1.svg:
Done in 4 ms!
3.016 KiB - 12.2% = 2.648 KiB

NegativeRank2.svg:
Done in 7 ms!
3.565 KiB - 14.3% = 3.057 KiB

2.svg:
Done in 3 ms!
3.926 KiB - 26.4% = 2.888 KiB

ArbitersLevel2.svg:
Done in 5 ms!
2.913 KiB - 12.6% = 2.545 KiB

ArbitersLevel3.svg:
Done in 2 ms!
1.725 KiB - 24% = 1.312 KiB

ArbitersLevel4.svg:
Done in 4 ms!
3.026 KiB - 14.7% = 2.58 KiB

kuva.svg:
Done in 2 ms!
0.944 KiB - 36.9% = 0.596 KiB

CephalonSudaLevel1.svg:
Done in 6 ms!
5.219 KiB - 7.8% = 4.813 KiB

ArbitersLevel5.svg:
Done in 6 ms!
3.812 KiB - 9.6% = 3.447 KiB

CephalonSudaLevel5.svg:
Done in 4 ms!
3.32 KiB - 13.3% = 2.878 KiB

EntratiLevel1.svg:
Done in 16 ms!
11.83 KiB - 6.9% = 11.01 KiB

kuvaflood.svg:
Done in 2 ms!
1.317 KiB - 31.4% = 0.904 KiB

Neutral.svg:
Done in 1 ms!
1.45 KiB - 23.6% = 1.107 KiB

CephalonSudaLevel2.svg:
Done in 9 ms!
6.109 KiB - 12.5% = 5.346 KiB

CephalonSudaLevel4.svg:
Done in 7 ms!
9.453 KiB - 8% = 8.698 KiB

EntratiLevel2.svg:
Done in 8 ms!
9.278 KiB - 4.6% = 8.855 KiB

EntratiLevel5.svg:
Done in 9 ms!
11.258 KiB - 5.7% = 10.616 KiB

NecraloidLevel1.svg:
Done in 20 ms!
16.351 KiB - 6.2% = 15.334 KiB

EntratiLevel3.svg:
Done in 9 ms!
10.982 KiB - 2.8% = 10.677 KiB

NecraloidLevel2.svg:
Done in 9 ms!
14.694 KiB - 1.4% = 14.493 KiB

CephalonSudaLevel3.svg:
Done in 6 ms!
10.246 KiB - 4.5% = 9.787 KiB

NewLokaLevel2.svg:
Done in 2 ms!
4.81 KiB - 7.1% = 4.467 KiB

NecraloidLevel3.svg:
Done in 12 ms!
14.603 KiB - 2.7% = 14.203 KiB

NewLokaLevel3.svg:
Done in 4 ms!
5.003 KiB - 6.5% = 4.678 KiB

NewLokaLevel1.svg:
Done in 12 ms!
13.551 KiB - 7% = 12.604 KiB

OstronLevel2.svg:
Done in 1 ms!
1.758 KiB - 22.6% = 1.361 KiB

EntratiLevel4.svg:
Done in 7 ms!
10.396 KiB - 7.5% = 9.611 KiB

NewLokaLevel5.svg:
Done in 5 ms!
7.263 KiB - 7.5% = 6.721 KiB

OstronLevel1.svg:
Done in 1 ms!
1.203 KiB - 28.5% = 0.86 KiB

OstronLevel5.svg:
Done in 3 ms!
4.021 KiB - 11.9% = 3.545 KiB

RedVeilLevel1.svg:
Done in 2 ms!
3.12 KiB - 8.9% = 2.842 KiB

OstronLevel4.svg:
Done in 3 ms!
3.417 KiB - 15% = 2.903 KiB

RedVeilLevel2.svg:
Done in 3 ms!
3.732 KiB - 7% = 3.471 KiB

NewLokaLevel4.svg:
Done in 2 ms!
5.237 KiB - 5.5% = 4.95 KiB

RedVeilLevel5.svg:
Done in 2 ms!
3.542 KiB - 7.8% = 3.265 KiB

OstronLevel3.svg:
Done in 2 ms!
2.597 KiB - 14% = 2.232 KiB

RedVeilLevel3.svg:
Done in 3 ms!
3.854 KiB - 8.2% = 3.537 KiB

SolarisLevel1.svg:
Done in 1 ms!
1.475 KiB - 32.5% = 0.996 KiB

SolarisLevel5.svg:
Done in 5 ms!
5.334 KiB - 13.4% = 4.62 KiB

SolarisLevel2.svg:
Done in 1 ms!
1.601 KiB - 29.5% = 1.128 KiB

SteelMeridianLevel1.svg:
Done in 1 ms!
1.812 KiB - 20.1% = 1.447 KiB

RedVeilLevel4.svg:
Done in 2 ms!
3.229 KiB - 9.9% = 2.908 KiB

SolarisLevel3.svg:
Done in 2 ms!
1.823 KiB - 25.5% = 1.357 KiB

SteelMeridianLevel5.svg:
Done in 1 ms!
1.399 KiB - 27.4% = 1.016 KiB

SteelMeridianLevel3.svg:
Done in 3 ms!
2.645 KiB - 10.8% = 2.358 KiB

HoldfastsLevel3.svg:
Done in 8 ms!
8.309 KiB - 4.9% = 7.9 KiB

SteelMeridianLevel2.svg:
Done in 10 ms!
8.321 KiB - 9.5% = 7.528 KiB

SolarisLevel4.svg:
Done in 2 ms!
2.6 KiB - 15.3% = 2.202 KiB

HoldfastsLevel2.svg:
Done in 6 ms!
9.271 KiB - 3.7% = 8.933 KiB

HoldfastsLevel1.svg:
Done in 6 ms!
8.104 KiB - 3.8% = 7.794 KiB

HoldfastsLevel5.svg:
Done in 9 ms!
10.657 KiB - 7.4% = 9.868 KiB

SteelMeridianLevel4.svg:
Done in 1 ms!
3.262 KiB - 7.7% = 3.011 KiB

PerrinLevel1.svg:
Done in 1 ms!
1.475 KiB - 24.4% = 1.115 KiB

PerrinLevel5.svg:
Done in 33 ms!
17.02 KiB - 8.9% = 15.505 KiB

PerrinLevel4.svg:
Done in 1 ms!
1.535 KiB - 29.3% = 1.086 KiB

HoldfastsLevel4.svg:
Done in 7 ms!
9.25 KiB - 5.3% = 8.759 KiB

PerrinLevel2.svg:
Done in 1 ms!
2.277 KiB - 17.9% = 1.87 KiB

QuillsLevel3.svg:
Done in 2 ms!
4.631 KiB - 4.4% = 4.426 KiB

VentKidsLevel1.svg:
Done in 12 ms!
14.439 KiB - 6.7% = 13.465 KiB

VentKidsLevel2.svg:
Done in 16 ms!
25.706 KiB - 6% = 24.167 KiB

PerrinLevel3.svg:
Done in 46 ms!
21.349 KiB - 12.6% = 18.658 KiB

QuillsLevel5.svg:
Done in 3 ms!
5.401 KiB - 4.5% = 5.158 KiB

QuillsLevel1.svg:
Done in 1 ms!
2.331 KiB - 11.9% = 2.054 KiB

QuillsLevel2.svg:
Done in 3 ms!
3.833 KiB - 6.4% = 3.589 KiB

QuillsLevel4.svg:
Done in 3 ms!
5.432 KiB - 3.3% = 5.252 KiB

VentKidsLevel5.svg:
Done in 32 ms!
34.704 KiB - 5.2% = 32.901 KiB

VoxLevel1.svg:
Done in 4 ms!
3.21 KiB - 16.2% = 2.691 KiB

VentKidsLevel3.svg:
Done in 11 ms!
16.556 KiB - 3.2% = 16.027 KiB

VoxLevel5.svg:
Done in 3 ms!
3.338 KiB - 19.7% = 2.679 KiB

VentKidsLevel4.svg:
Done in 27 ms!
38.733 KiB - 4.2% = 37.1 KiB

VoxLevel2.svg:
Done in 5 ms!
2.473 KiB - 20.6% = 1.963 KiB

VoxLevel4.svg:
Done in 4 ms!
2.718 KiB - 19.1% = 2.199 KiB

VoxLevel3.svg:
Done in 5 ms!
3.904 KiB - 16.5% = 3.262 KiB
```

**Comment by SlayerOrnstein** [comment]
Used https://github.com/twbs/fantasticon to generate an icon font called `Warframe-symbols`. Code points can be found in `font/Warframe-symbols.json` and a CSS was generated with classes setup for each icon along with a types export in typescript

`fantasticon svg/ -n "Warframe-symbols" --normalize true --output fonts/` 

Make sure to use `--normalize true` to set all SVGs to the same height in the icon font. It will use the height of the biggest SVG

File: scripts/batch-png-to-svg.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# otherwise you could have misalinged lines and basically blobs
+
+# Loop through PNG files
+for file in *.png; do
+  # Extract filename without extension
+  basename=$(echo "$file" | sed 's/\.\w*$//')
+
+ # Convert PNG to black PNG with transparency
+ # We need to convert the image from black to white so potrace can detect
+ # and trace properly
```
Comment (TobiTenno) [inline scripts/batch-png-to-svg.sh]: needs another space at the front

```suggestion
  # Convert PNG to black PNG with transparency
  # We need to convert the image from black to white so potrace can detect
  # and trace properly
```

Comment (SlayerOrnstein) [reply scripts/batch-png-to-svg.sh]: Wouldn't worry too much about this script, I'll be actively looking for alternatives

**Review by TobiTenno — APPROVED** [review]

**Pull request merged**